middleware/cache/cache_test.go

```diff
@@ -345,6 +345,38 @@ func Test_CustomExpiration(t *testing.T) {
 	}
 }
 
+func Test_NothingToCache_CustomExpiration(t *testing.T) {
+	t.Parallel()
+
+	app := fiber.New()
+
+	app.Use(New(Config{ExpirationGenerator: func(c *fiber.Ctx, cfg *Config) time.Duration {
+		newCacheTime, _ := strconv.Atoi(c.GetRespHeader("Cache-Time", "600"))
+		return time.Second * time.Duration(newCacheTime)
+	}}))
+
+	app.Get("/", func(c *fiber.Ctx) error {
+		c.Response().Header.Add("Cache-Time", "-1")
+		return c.SendString(time.Now().String())
+	})
+
+	resp, err := app.Test(httptest.NewRequest("GET", "/", nil))
+	utils.AssertEqual(t, nil, err)
+	body, err := ioutil.ReadAll(resp.Body)
```

Comment:
```suggestion
	body, err := io.ReadAll(resp.Body)
```
see this https://go.dev/doc/go1.16#ioutil

Reply:
it's breaking change for 1.14 and 1.15